docs/index.md

```diff
@@ -3,174 +3,26 @@
 page_title: "ocm Provider"
 subcategory: ""
 description: |-
```

Comment:
I prefer you'll chackout those changes in the index.md - 
@datianshi  edit that file manually